manifest_staging/charts/secrets-store-sync-controller/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: secrets-store-sync-controller
 version: 0.0.1
-appVersion: v0.0.1
+appVersion: 0.0.1
```

Comment:
```suggestion
appVersion: v0.0.1
```

Reply:
I'm assuming you're doing the same as driver. For driver the versioning didn't have the `v` initially and after it wasn't possible to change the app version. For the controller, let's start using the correct app version.

Reply:
Yeah. I'll update promote target to reflect this.